bin/dispatch-conf

```diff
@@ -168,8 +165,9 @@ class dispatch:
             try:
                 path_list = _unicode_decode(
                     subprocess.check_output(
+                        # Find existing configs
                         portage.util.shlex_split(
-                            FIND_EXTANT_CONFIGS % (path, find_opts, basename)
+                            f"find '{path}' {find_opts} -name '._cfg????_{basename}' ! -name '.*~' ! -iname '.*.bak' -print"
```

Comment:
It looks like `shlex.split()` does shell-style word splitting which is almost just a "split this string on spaces" here. The `find_opts` makes it slightly trickier since it can be either `"-name '.*' -type d -prune -o"` or `"-maxdepth 1"`.

Would something like this work?
```python
subprocess.check_output(
  ["find", path]
  + shlex_split(find_opts)
  + ["-name", ...]
)
```

Reply:
I would would change `find_opts` to be a list of strings in the first place, instead of a concatenating a shell command and then splitting it using shlex.

Reply:
That's a much better plan.